src/main/java/com/crowdin/client/sourcestrings/model/SourceStringForm.java

```diff
@@ -0,0 +1,16 @@
+package com.crowdin.client.sourcestrings.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class SourceStringForm {
```

Comment:
is this form used only in tests?

Reply:
> is this form used only in tests?

Yes. Initially I tried to adapt `SourceString` model for it but failed. The original model has some fields of primitive types, which have default value and always serialized. This model can also be used in production without breaking changes